blog/2023-06-23-newsletter.mdx

```diff
@@ -0,0 +1,164 @@
+
+## Rattler!
+
+<Image img={rattler} alt="Banner image for Rattler, from GitHub project"/>
+
+**[rattler](https://github.com/mamba-org/rattler) is a new collection of Rust libraries to work with conda packages made by the [prefix.dev](https://prefix.dev/) team!**
+
+* rattler 0.3.0 just came out with lots of highlights, such as JLAP support, local `file://` urls and authenticated requests with the new `rattler_networking`. See the full ( and impressive) [changelog](https://github.com/mamba-org/rattler/releases/tag/v0.3.0) for more.
+
+**[rattler-build](https://prefix.dev/blog/the_new_rattler_build)** is built on top rattler to facilitate the creation of conda packages. It already _works_ very well and is much faster than conda-build. To push this effort forward, we started multiple conda enhancement proposals. Engage with us and the conda community on a forward-looking YAML format that is easy to parse (pure YAML) and keeps most of the conda-build functionalities!
+
+* A [new YAML format](https://github.com/conda-incubator/ceps/pull/54).
+* Integration with VSCode for the [new YAML format with automatic syntax linting](https://github.com/conda-incubator/ceps/pull/56#issuecomment-1593118521).
+* A new way to [define multiple outputs and tests](https://github.com/conda-incubator/ceps/pull/56).
+* A new test for [file existence inside of packages](https://github.com/conda-incubator/ceps/pull/57).
+
+---
+
+## Community channel updates
+
+[<Image img={cf} alt="conda-forge logo, with text"/>](https://conda-forge.org/)
+
+### conda-forge
+
+**conda-forge has a Google Summer of Code project to update the [project website](https://conda-forge.org/)**. [Asmit Malakannnawar](https://asmitbm.github.io/) will be working on the update for the next few months. The update will provide an intuitive and usable user experience that also passes SpeedPage and [WAVE](https://wave.webaim.org/) tests, and that moves the site closer to [WCAG 2](https://www.w3.org/WAI/standards-guidelines/wcag/) conformance. He'll also produce a site style guide. Have ideas? [Reach out to Asmit](https://asmit.bio.link/).
+
+---
+
+[<Image img={bioconda} alt="Bioconda logo"/>](https://bioconda.github.io/)
+
+### Bioconda
+
+The [Bioconda project](https://bioconda.github.io/) has conducted a major rebuild of its packages, updating to the latest dependency pinnings of conda-forge. This entails using gcc 12 on linux and clang 15 on macOS, as well as using openssl 3. While most packages could be updated flawlessly, there are various remainders that are currently skiplisted. *Your help in fixing them would be greatly appreciated.* Just choose a recipe from **[this list](https://github.com/bioconda/bioconda-recipes/wiki/build-failures)**, check the provided build logs and create a PR with your solution to fix the failures. Sometimes, it could be as easy as updating to the latest upstream version.
```

Comment:
No idea what skiplisted means. Is that a common term?

Reply:
Apparently "[skiplist](https://opendatastructures.org/ods-java/4_Skiplists.html)" is a common term, but it doesn't line up with how it used here!.  I will reword in next commit.